src/ex6/server/client/src/components/AddTask/AddTask.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from "react"
+import "./style.css"
+import { ItemClient } from "./item_client"
+import deleteIcon from "./images/delete_icon.svg"
+
+export const AddTask = () => {
+  const itemClient = new ItemClient()
+  const [task, setTask] = useState("")
+  const [tasks, setTasks] = useState([])
+  const [isLoading, setIsLoading] = useState(false)
+  const [error, setError] = useState(null)
+
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
you should define the method fetchData outside the useEffect block, then just call it